runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java

```diff
@@ -218,6 +240,38 @@ private void onClanMemberLeft(ClanMemberLeft event)
 		getCallerList();
 	}
 
+	private void onPlayerSpawned(PlayerSpawned event)
+	{
+		final Player player = event.getPlayer();
+
+		if (player == client.getLocalPlayer())
+		{
+			return;
+		}
+
+		if (checkWildy() && isShowAgilityLevel())
+		{
+			if (!hiscoreCheckBlacklist.isEmpty() && hiscoreCheckBlacklist.keySet().contains(player.getName()))
```

Comment:
remove `isEmpty()`
if `hiscoreCheckBlacklist.getOrDefault(player.getName(), 1) < client.getTickCount()`

Reply:
I switched to using HiscoreManager which already does the caching for me